packages/block-editor/src/hooks/border.js

```diff
@@ -24,15 +27,18 @@ export function BorderPanel( props ) {
 
 	return (
 		<InspectorControls>
-			<PanelBody title={ __( 'Border settings' ) }>
+			<PanelBody title={ __( 'Border settings' ) } initialOpen={ false }>
```

Comment:
It seems like we can simplify quite a bit the code across this file and the `border-*.js` files if we do (pseudo-code):

```js
const hasSupport = hasBorderSupport( props );

const isColorSupported  = ! useEditorFeature( 'border.customColor' );
const isRadiusSupported = ! useEditorFeature( 'border.customRadius' );
const isStyleSupported  = ! useEditorFeature( 'border.customStyle' );
const isWidthSupported  = ! useEditorFeature( 'border.customWidth' );

if ( ! hasSupport || allAreDisabled ) {
	return null;
}
  

return (
	<PanelBody >
		isBorderSupported && <BorderStyleEdit />
		isWidthSupported && <BorderWidthEdit />
		isRadiusSupported && <BorderRadiusEdit />
		isColorSupported && <BorderColorEdit />
	</PanelBody>
);
```

Reply:
Thanks for the suggestion. I've made these changes ✅